plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -707,6 +709,23 @@ private String setupMountPoint(String parent) {
         return mountPoint;
     }
 
+    @Override
+    public synchronized void setSystemVmTmpltPermission(String mountPoint) {
+        if (!s_isSystemVmTmpltPermissionSet) {
+            s_logger.debug("Set permissions for " + mountPoint);
+            String result = null;
+            final String systemVmTmpltPermissions = "0777";
+            Script script = new Script(true, "chmod", _timeout, s_logger);
+            script.add("-R", systemVmTmpltPermissions, mountPoint);
```

Comment:
@SudharmaJain Used the string as suggested instead of hardcoded value for permissions.